src/orderbook.ts

```diff
@@ -266,11 +297,16 @@ function priceToCoverAmount(amount: Fraction, offers: Offer[]) {
   return undefined;
 }
 
-function invertPricePoints(prices: Map<number, Offer>) {
+function invertPricePoints(
+  prices: Map<number, Offer>,
+  volumePriceAdjustmentForFee: Fraction
+) {
   return new Map(
     Array.from(prices.entries()).map(([_, offer]) => {
       const inverted_price = offer.price.inverted();
-      const inverted_volume = offer.volume.mul(offer.price);
+      const inverted_volume = offer.volume
```

Comment:
Are you sure this is correct?

I would have guessed: 
```
const inverted_volume = offer.volume.div(volumePriceAdjustmentForFee)
        .mul(offer.price)
        .mul(volumePriceAdjustmentForFee);```
=>
```
const inverted_volume = offer.volume.
        .mul(offer.price);
```
You can easily convince me by adding a unit test for idempotence on inversion.

Reply:
They are not the same.

The problem is that we need to multiply the volume with the price before fee. Thus we multiply offer.price with volumePriceAdjustmentForFee and then multiply that with volume.

```
const inverted_volume = offer.volume.
        .mul(price_without_fee);
```

Will change the code to reflect that thought process better.